lib/components/Masthead/Masthead.tsx

```diff
@@ -103,122 +110,110 @@ const StyledButton = styled(Attr.button)`
-                    label={search.label}
-                    attr={search.attr}
-                />}
-                <ThemeProvider theme={toolbarButtonTheme}>
-                    <Attr.div className={cx('masthead-toolbar-container', { 'force-hide-search': expanded })}>
-                        <ul className={cx('masthead-toolbar')}>
-                            {search && <li key='item-search' className={cx('search-button')}>
-                                <StyledButton
-                                    key={search.label}
-                                    attr={{ button: { title: search.label } }}
-                                    onClick={search.onExpand}
-                                    className={cx('masthead-btn')}>
-                                        <Icon icon='search' size={IconSize.xsmall} />
-                                </StyledButton>
-                            </li>}
-                            {more && !more.selected && items}
-                            {more &&
-                                <li key='item-more' className={cx('more-button')} title={more.attr?.ariaLabel}>
-                                    <InlinePopup.Container
-                                        expanded={more.selected}
-                                        onClick={more.onClick}
-                                        attr={more.attr}>
-                                        <StyledButton
-                                            attr={more.attr}
-                                            title={more.title}
-                                            onClick={more.onClick}
-                                            className={cx('masthead-btn', 'more-menu-btn', { 'selected': more.selected })}>
-                                                <Icon icon='more' size={IconSize.xsmall} />
-                                        </StyledButton>
-                                        <InlinePopup.Panel
-                                            alignment='right'
-                                            className={cx('masthead-toolbar-menu')}
-                                        >
-                                            <ul role='menu' id='more-menu'>
-                                                {items}
-                                            </ul>
-                                        </InlinePopup.Panel>
-                                    </InlinePopup.Container>
-                                </li>
-                            }
-                            {user && <li key='user-menu' className={cx('user-menu-item')}>
-                                {user}
-                            </li>}
-                        </ul >
-                    </Attr.div>
-                </ThemeProvider>
-            </MastheadContainer>
-        );
-    }
-}
+                        </li>}
+                        {more && !more.selected && items}

```

Comment:
this seems weird to me. are we actually saying "show items if more exists yet we didn't select it"? or are we trying to say something else?

Reply:
yeah that it is exactly what it is saying haha... the thing is that the masthead on smaller screens hides the items and shows a chevron so you can open up the items in a `more` popup menu.... so this logic is there to make that hidding possible